app/src/components/settings/RolePicker.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+import { Item, Picker, PickerProps } from "@arizeai/components";
+
+import { UserRole } from "@phoenix/constants";
+
+const UserRoles = Object.values(UserRole);
```

Comment:
I'd sorta prefer to make this use the API to get the roles. It might be some overhead right now but ultimately a tad more flexible and honest

Reply:
That works for me, i can make a follow up to do that